main.js

```diff
@@ -0,0 +1,162 @@
+}
+
+initDragDrop();
+
+function onMouseDown(e) {
+    // IE doesn't pass the event object
+    if (e == null) 
+        e = window.event; 
+    
+    // IE uses srcElement, others use target
+    var target = e.target != null ? e.target : e.srcElement;
+    
+    // for IE, left click == 1
+    // for Firefox, left click == 0
+    if ((e.button == 1 && window.event != null || e.button == 0) && target.className.indexOf('drag') > -1) {
+        // grab the mouse position
+        _startX = e.clientX;
+        _startY = e.clientY;
+        
+        // grab the clicked element's position
+        _offsetX = extractNumber(target.style.left);
+        _offsetY = extractNumber(target.style.top);
+        
+        // bring the clicked element to the front while it is being dragged
+        iterator++;
+        target.style.zIndex = iterator;
+        
+        // we need to access the element in OnMouseMove
+        _dragElement = target;
+
+        // tell our code to start moving the element with the mouse
+        document.addEventListener('mousemove', onMouseMove);
+
+
+        // prevent text selection in IE
+        document.onselectstart = function () { return false; };
+        // prevent IE from trying to drag an image
+        target.ondragstart = function() { return false; };
+
+        document.addEventListener('mouseup', onMouseUp);
+        document.removeEventListener('mousedown', onMouseDown);
+        
+        // prevent text selection (except IE)
+        return false;
+     }
+}
+
+var onMouseMove = function(e) {
+    if (e == null) 
+        var e = window.event; 
+
+    // this is the actual "drag code"
+    _dragElement.style.left = (_offsetX + e.clientX - _startX) + 'px';
+    _dragElement.style.top = (_offsetY + e.clientY - _startY) + 'px';
+     
+}
+
+function onMouseUp(e) {
+
+    if (_dragElement != null)
+    {
+        // _dragElement.style.zIndex = ;
+
+        // we're done with these events until the next OnMouseDown
+        document.removeEventListener('mousemove', onMouseMove);
+        document.addEventListener('mousedown', onMouseDown);
+        document.removeEventListener('mouseup', onMouseUp);
+
+        document.onselectstart = null;
+        _dragElement.ondragstart = null;
+
+        // this is how we know we're not dragging      
+        _dragElement = null;
+    }
+}
+
+function extractNumber(value) {
+    var n = parseInt(value);
+	
+    return n == null || isNaN(n) ? 0 : n;
+}
+
+
+function convertInput(inputTextValue, selectValue) {
+    var selectInput = document.getElementById('selectExisting');
+    var selectValue = selectInput.options[selectInput.selectedIndex].value;
+    var inputTextValue = $('#inputTextArea').val().trim();
+
+    console.log(inputTextValue, "text");
+
+    //if selection & value text box, reset select and process the text box
+    if (inputTextValue.length > 1) {
+        selectInput.value = 'selectone';
+        var splitInput = inputTextValue.split(",");
```

Comment:
you should declare splitInput at the beginning of the function and not inside the if/else